src/components/shared/Console.js

```diff
@@ -38,7 +38,7 @@ export default function Console({ executable, output }) {
 
   return (
     <ConsoleWrapper>
-      <DemoText paddingBottom={'5px'} onClick={toggleDetails}>
+      <DemoText marginBottom={'5px'} onClick={toggleDetails}>
```

Comment:
why ConsoleWrapper can't have padding?

Reply:
that change is only because I changed `DemoText` to use `margin` rather than `padding`

Reply:
it overrides `margin: 0 0 25px 0;`